FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthProvider.swift

```diff
@@ -22,6 +22,10 @@ import Foundation
 @objc(FIRPhoneAuthProvider) open class PhoneAuthProvider: NSObject {
   /// A string constant identifying the phone identity provider.
   @objc public static let id = "phone"
+  @objc private static let kRecaptchaVersion = "RECAPTCHA_ENTERPRISE"
+  @objc private static let kClientType = "CLIENT_TYPE_IOS"
+  @objc private static let kFakeCaptchaResponse = "NO_RECAPTCHA"
```

Comment:
`@objc` in combination with `private` should be a no-op, and `k` prefixes in Swift are not needed.

```suggestion
  private static let recaptchaVersion = "RECAPTCHA_ENTERPRISE"
  private static let clientType = "CLIENT_TYPE_IOS"
  private static let fakeCaptchaResponse = "NO_RECAPTCHA"
```

Reply:
Addressed in https://github.com/firebase/firebase-ios-sdk/pull/14170